packages/client/core/src/api/simpleCryptidClient.ts

```diff
@@ -17,7 +17,7 @@ export class SimpleCryptidClient extends AbstractCryptidClient {
     return this._wallet;
   }
 
-  as(controlledDid: string): ControlledCryptidClient {
-    return new ControlledCryptidClient(controlledDid, this, this.options);
+  controlWith(controllerDid: string): CryptidClient {
```

Comment:
actually found the `as` quite elegant, but understand that this is more descriptive.